shared/src/iosMain/kotlin/co/touchlab/droidcon/MainScope.kt

```diff
@@ -2,12 +2,13 @@ package co.touchlab.droidcon
 
 import co.touchlab.droidcon.util.printThrowable
 import co.touchlab.kermit.Logger
+import kotlin.coroutines.CoroutineContext
 import kotlinx.coroutines.CoroutineExceptionHandler
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.SupervisorJob
-import kotlin.coroutines.CoroutineContext
 
-class MainScope(private val mainContext: CoroutineContext, private val log: Logger) : CoroutineScope {
+class MainScope(private val mainContext: CoroutineContext, private val log: Logger) :
```

Comment:
Is there a way to configure it to put the parameters to new lines instead of putting the inheritance onto the new line?